test/files/neg/t10279.scala

```diff
@@ -0,0 +1,10 @@
+object Test {
+
+  def foo(i: Int)(implicit s: String): String = ???
+
+  def test(implicit s: String) {
+    // foo(1) _
+  }
+
+  foo(1) _
```

Comment:
let's add the slightly simpler test case from https://github.com/scala/bug/issues/10299
```
scala> def foo(implicit x: Int): Int = x
scala> val bar = foo _
```